src/RichTextEditor/FormattingToolbar/HyperlinkModal.js

```diff
@@ -219,25 +224,49 @@ const HyperlinkModal = React.forwardRef(({ ...props }, ref) => {
               </InlineFormButton>
           </InlineFormField>
           <InlineFormField>
-              <LinkIconHolder
-                onClick={copyLink}
-                aria-label="Copy hyperlink text"
-              >
-                <CopyIcon />
-              </LinkIconHolder>
-              <LinkIconHolder
-                disabled={!isSelectionLink(editor)}
-                onClick={removeLink}
-                aria-label="Remove hyperlink"
-              >
-                <DeleteIcon />
-              </LinkIconHolder>
-              <LinkIconHolder
-                onClick={openLink}
-                aria-label="Open hyperlink"
-              >
-                <OpenIcon />
-              </LinkIconHolder>
+              <Popup
+                trigger={
+                  <LinkIconHolder
+                    onClick={copyLink}
+                    aria-label="Copy hyperlink text"
+                  >
+                    <CopyIcon />
+                  </LinkIconHolder>
+                }
+                content="Copy hyperlink text"
+                inverted
+                position='bottom left'
+                style={popupStyles}
+              />
+              <Popup
+                trigger={
+                  <LinkIconHolder
+                    disabled={!isSelectionLink(editor)}
```

Comment:
Looks like this already existed from the previous PR, but what does `disabled={!isSelectionLink(editor)}` do here? I don't see any UI difference.

Reply:
@irmerk this was used when a button was being used for the 'remove' option. As we are not using it now we can remove this.
Also can I use
```js 
onClick={isSelectionLink(editor) ? removeLink : null}
```
on the delete iconHolder so that it can't be used if the selection is not already a link i.e. one would not be able to use this while creating a link ?

Reply:
Yeah I think that would be good to do.

Reply:
@irmerk Just made the changes. Please review!
Thank You